app/controllers/reports_controller.rb

```diff
@@ -0,0 +1,42 @@
+class ReportsController < ApplicationController
+  def index
+    if current_instructor
+      @reports = Report.all
+    else
+      @reports = Report.where(student_id: current_student.id)
+    end
+  end
+
+  def new
+    authenticate_instructor!
```

Comment:
At the moment I have Reports up at `/reports` because WIP. I want students to be able to see their own reports.